<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>798</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>9</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>高级绘图模式</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1,1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>X轴名称</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="xlabel">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;
</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>读取位置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="xline">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
          </property>
          <property name="placeholderText">
           <string>无则不填</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="xinput">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(240, 255, 247);
border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
        </property>
        <property name="placeholderText">
         <string>请在此处输入横坐标数据（空格分隔）</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Y轴名称</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ylabel">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>读取位置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="yline">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
          </property>
          <property name="placeholderText">
           <string>无则不填</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="yinput">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(244, 255, 234);
border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
        </property>
        <property name="placeholderText">
         <string>请在此处输入纵坐标数据（空格分隔）</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="legendlayout">
        <item>
         <widget class="QLabel" name="labell">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>本图像名称</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="labelinput">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
          </property>
          <property name="placeholderText">
           <string>请输入第1幅图的名称</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="history_layout">
        <item>
         <widget class="QPushButton" name="hisupbtn">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(221, 226, 255);
border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
          </property>
          <property name="text">
           <string>上条历史记录</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="hisdownbtn">
          <property name="font">
           <font>
            <family>Adobe Devanagari</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(221, 226, 255);
border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
          </property>
          <property name="text">
           <string>下条历史记录</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearbtn">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(246, 255, 237);
border:2px groove gray;border-radius:10px;padding:2px 4px;
</string>
          </property>
          <property name="text">
           <string>归零</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,3,0,4,0,0,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Z轴名称</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="zlabel">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="zlinelabel">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>读取位置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="zline">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
          </property>
          <property name="placeholderText">
           <string>无则不填</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="zinput">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 250, 229);
border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
        </property>
        <property name="placeholderText">
         <string>请在此处输入Z轴的值（空格分隔）</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="titlelable">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>图表标题</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="title">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="numinputlabel">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>曲线条数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="numinput">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
          </property>
          <property name="placeholderText">
           <string>1可不填</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="image_show">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 250, 229);
border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7"/>
      </item>
      <item>
       <widget class="QPushButton" name="drawbtn">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 255, 255);
border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
        </property>
        <property name="text">
         <string>开始绘制</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>842</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="txtfile"/>
    <addaction name="excelfile"/>
    <addaction name="export_image"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="helpdocu"/>
    <addaction name="updatedocu"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>可选功能</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>坐标旋转</string>
     </property>
     <addaction name="d10"/>
     <addaction name="d20"/>
     <addaction name="d30"/>
     <addaction name="d45"/>
     <addaction name="d50"/>
     <addaction name="d60"/>
     <addaction name="d75"/>
     <addaction name="d80"/>
     <addaction name="d90"/>
    </widget>
    <addaction name="grid"/>
    <addaction name="data_label"/>
    <addaction name="legend"/>
    <addaction name="menu_4"/>
    <addaction name="img_sep_show"/>
   </widget>
   <widget class="QMenu" name="menuexcel">
    <property name="title">
     <string>Excel自选功能</string>
    </property>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>横竖读法</string>
     </property>
     <addaction name="readmethod1"/>
     <addaction name="readmethod2"/>
    </widget>
    <addaction name="menu_5"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>高级绘图选项</string>
    </property>
    <addaction name="dscatter"/>
    <addaction name="dplot"/>
    <addaction name="d3image"/>
   </widget>
   <widget class="QMenu" name="menu_7">
    <property name="title">
     <string>图片个数</string>
    </property>
    <addaction name="num12"/>
    <addaction name="num13"/>
    <addaction name="num21"/>
    <addaction name="num22"/>
    <addaction name="num23"/>
    <addaction name="num31"/>
    <addaction name="num32"/>
    <addaction name="num33"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menuexcel"/>
   <addaction name="menu_6"/>
   <addaction name="menu_7"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="txtfile">
   <property name="text">
    <string>打开文本文件</string>
   </property>
  </action>
  <action name="excelfile">
   <property name="text">
    <string>打开Excel文件</string>
   </property>
  </action>
  <action name="helpdocu">
   <property name="text">
    <string>使用说明</string>
   </property>
  </action>
  <action name="updatedocu">
   <property name="text">
    <string>更新日志</string>
   </property>
  </action>
  <action name="grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>网格线</string>
   </property>
  </action>
  <action name="data_label">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>数据标签</string>
   </property>
  </action>
  <action name="legend">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>图例</string>
   </property>
  </action>
  <action name="d10">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>10°</string>
   </property>
  </action>
  <action name="d20">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>20°</string>
   </property>
  </action>
  <action name="d30">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>30°</string>
   </property>
  </action>
  <action name="d45">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>45°</string>
   </property>
  </action>
  <action name="d50">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>50°</string>
   </property>
  </action>
  <action name="d60">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>60°</string>
   </property>
  </action>
  <action name="d75">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>75°</string>
   </property>
  </action>
  <action name="d80">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>80°</string>
   </property>
  </action>
  <action name="d90">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>90°</string>
   </property>
  </action>
  <action name="img_sep_show">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>图片单独显示</string>
   </property>
  </action>
  <action name="export_image">
   <property name="text">
    <string>导出图片</string>
   </property>
  </action>
  <action name="readmethod1">
   <property name="text">
    <string>横向读取</string>
   </property>
  </action>
  <action name="readmethod2">
   <property name="text">
    <string>纵向读取</string>
   </property>
  </action>
  <action name="dscatter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>三维散点图</string>
   </property>
  </action>
  <action name="dplot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>三维曲线图</string>
   </property>
  </action>
  <action name="d3image">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>三维曲面图</string>
   </property>
  </action>
  <action name="num12">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1X2</string>
   </property>
  </action>
  <action name="num13">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1X3</string>
   </property>
  </action>
  <action name="num21">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2X1</string>
   </property>
  </action>
  <action name="num22">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2X2</string>
   </property>
  </action>
  <action name="num23">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2X3</string>
   </property>
  </action>
  <action name="num31">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3X1</string>
   </property>
  </action>
  <action name="num32">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3X2</string>
   </property>
  </action>
  <action name="num33">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3X3</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
